Dockerfile

```diff
@@ -17,12 +17,12 @@ RUN go get github.com/golang/protobuf/proto && \
 COPY --from=haskell /root/.cabal/bin/proto-lens-protoc /usr/local/bin/proto-lens-protoc
 
 # Bit of a hack so that proto-lens-protoc actually runs
-COPY --from=haskell /opt/ghc/8.8.3/lib/ghc-8.8.3/* /opt/ghc/8.8.3/lib/ghc-8.8.3/
+COPY --from=haskell /opt/ghc/8.8.1/lib/ghc-8.8.1/* /opt/ghc/8.8.1/lib/ghc-8.8.1/
```

Comment:
When pairing, @tclem and found that `protoc` did not work with ghc 8.8.3. We didn't investigate the error too deeply though.